### PR TITLE
Remove deprecated create_function

### PR DESCRIPTION
**Pull request by heiglandreas**
This replaces the deprecated `create_function` with an anonymous function.

### CODE REVIEW

File: markitdownwordpress.php [hunk 1]
```diff
@@ -33,7 +33,7 @@
     /*
      * Disable the wordpress visual editor, in Markdown writing we don't need it
      */
-    add_filter('user_can_richedit', create_function('', 'return false;'), 50);
+    add_filter('user_can_richedit', function() {return false;}, 50);
```
Comment (bueltge) [inline markitdownwordpress.php]: I think you should use the core fuction `__return_false` so you can reduce this hook to
`add_filter('user_can_richedit', '__return_false', 50);`

**Review by bueltge — CHANGES_REQUESTED** [review]

**Review by bueltge — APPROVED** [review]

**Comment by bueltge** [comment]
Wow, faster commit as my comment ;)

**Comment by heiglandreas** [comment]
:joy: Well, your comment and the approval wasn't slow either ;-)

**Comment by wkhayrattee** [comment]
thanks guys!
I will push this change on the wordpress plugin repo

**Pull request merged**